src/components/input-panel/spec-editor/index.js

```diff
@@ -8,7 +8,6 @@ const mapStateToProps = function (state, ownProps) {
     mode: state.app.mode,
     selectedExample: state.app.selectedExample,
     gist: state.app.gist,
-    compiledVegaSpec: state.app.compiledVegaSpec,
```

Comment:
This doesn't. I just removed it because compiledVegaSpec is not used in spec-editor/render.js